<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCad_Usuario</class>
 <widget class="QDialog" name="frmCad_Usuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro</string>
  </property>
  <property name="windowIcon">
   <iconset resource="arquivos.qrc">
    <normaloff>:/Arquivos/Imagens/usuario.png</normaloff>:/Arquivos/Imagens/usuario.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="5" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="clienteLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cliente :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="clienteComboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>1  -  Bibliotecario</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2  -  Professor</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3  -  Aluno</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="telefoneLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Telefone</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_AdicionarTelefone">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid grey;
text-align: left; </string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="arquivos.qrc">
         <normaloff>:/Arquivos/Imagens/adicionar.png</normaloff>:/Arquivos/Imagens/adicionar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="enderecoLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Endereco</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_AdicionarEndereco">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid grey;
text-align: left; </string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="arquivos.qrc">
         <normaloff>:/Arquivos/Imagens/adicionar.png</normaloff>:/Arquivos/Imagens/adicionar.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancelar">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="arquivos.qrc">
         <normaloff>:/Arquivos/Imagens/cancel.png</normaloff>:/Arquivos/Imagens/cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cadastrar">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Salvar</string>
       </property>
       <property name="icon">
        <iconset resource="arquivos.qrc">
         <normaloff>:/Arquivos/Imagens/OK.png</normaloff>:/Arquivos/Imagens/OK.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="cPFLabel">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CPF  :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="cPFLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="nomeLabel">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nome :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="nomeLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="emailLabel">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Email :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="emailLineEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>clienteComboBox</tabstop>
  <tabstop>cPFLineEdit</tabstop>
  <tabstop>nomeLineEdit</tabstop>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>pushButton_AdicionarTelefone</tabstop>
  <tabstop>pushButton_AdicionarEndereco</tabstop>
  <tabstop>pushButton_Cadastrar</tabstop>
  <tabstop>pushButton_Cancelar</tabstop>
 </tabstops>
 <resources>
  <include location="arquivos.qrc"/>
 </resources>
 <connections/>
</ui>
